### DISCUSSION TITLE
Combining samples

### DISCUSSION THREAD

**Discussion by saramoein372** [tuonglab/dandelion#107] [General]
Hi Kelvin,

I have a question about generating networks based on combining samples. The way I am doing is concatenating the adata files and also concatenating the VDJ files. But the final results looks something is wrong, because the obtained edges are much less than the individual samples.
Is there any example about it? Or any instruction for combining the samples to generating the networks?

Thank you,
Sara

**Comment by zktuong** [comment]
Hi Sara, when you concatenated the anndata object, were barcodes changed?

You can take a look here for how i used a combined object to run the networks.
https://nbviewer.org/github/scCOVID-19/COVIDPBMC/blob/master/B_cell/Analysis_and_plotting/BCR_network/network/021821_NewcastleSangerCambridge_BCR_bcells_bcr_network.ipynb

the gist is the same though. Without seeing the plots, it's hard for me to diagnose what's wrong.
